README.md

```diff
@@ -13,7 +13,7 @@ LORIS (Longitudinal Online Research and Imaging System) is a self-hosted web app
 This Readme covers installation of LORIS version <b>23</b> on <b>Ubuntu</b>.
 ([CentOS Readme also available](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md)).
 
-Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+Please consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).
```

Comment:
Should I include something along the lines of:

> Please consult the LORIS Installation Markdowns **located in the /docs/wiki directory**.

Reply:
if we can avoid the confusing path `docs/wiki/` that would be nice, to minimize changes down the road

Reply:
note: [alex's comment below](https://github.com/aces/Loris/pull/6712#issuecomment-642218561) was in response to this thread